src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -165,6 +165,14 @@ public static bool IsContainer(this IEnvironment environment)
                 && runningInContainerValue;
         }
 
+        public static bool IsCorsConfigurationAllowed(this IEnvironment environment)
+        {
+            var allowCorsConfiguration = environment.GetEnvironmentVariable(AllowCorsConfiguration);
+            return !string.IsNullOrEmpty(allowCorsConfiguration)
+                && bool.TryParse(allowCorsConfiguration, out bool allowCorsConfigurationValue)
```

Comment:
In other places, we look for the value "1" for truthiness. Should we stick to that approach?

https://github.com/Azure/azure-functions-host/blob/41b15ce28d4a012a4118840332a2cdd2b88ced85/src/WebJobs.Script/Environment/EnvironmentExtensions.cs#L50-L57